csfieldguide/static/interactives/persona-carousel/js/persona-carousel.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Define all personas in this dict
+ */
+var PERSONAS = {
+  bevan: {
+    carousel_id: 'bevan',
+    name: 'Bevan',
+    age: 21,
+    title: gettext('Accounting student at university'),
+    about: gettext('Bevan is a busy undergraduate student.\
+      He gets good grades and plays in a social football league.\
+      He is aware that he probably isn’t getting enough sleep, particularly as the exam period is coming up and he’s staying up late most nights to study.\
+      He knows that if he could improve the quality of the sleep that he is getting he would be more productive when studying and have more energy for his football games and training.'),
+    goals: [
+      gettext('Wake up feeling rested.'),
+      gettext('Be more productive during the day rather than feeling slow and sluggish.')
+    ],
+    frustrations: [
+      gettext('Feels that most nights when he finally gets to bed he lies awake for a while.'),
+      gettext('His exhaustion is affecting his performance in sport and education.'),
+      gettext('Doesn’t know where to start with improving his sleep habits.'),
+      gettext('Feels that he can only have two out of sleep, football and good grades.')
+    ]
+  },
+
+  sam: {
+    carousel_id: 'sam',
+    name: 'Sam',
+    age: 17,
+    title: gettext('High school student'),
+    about: gettext('Sam forgot to look both ways, and her subsequent injury has left her largely bound to a wheelchair.\
```

Comment:
Consensus in the office is that 

> Sam forgot to look both ways, and her subsequent injury

makes Sam look bad. While it can be factual, it brings shame to someone. A better phrase would be:

> Sam has suffered a spinal injury, and her injury